src/lib/parsers/general.ts

```diff
@@ -134,17 +199,19 @@ export interface GeneralRet {
   timePlayed?: string;
   farmInfo?: string;
   totalMoneyEarned?: number;
-  skills?: Record<Skill, number>;
+  skills?: Skills;
   questsCompleted?: number;
-  stardrops?: string[];
-  experience?: Record<Skill, number>;
+  stardrops?: Stardrops;
+  experience?: Experience;
   gameVersion?: string;
```

Comment:
Convert to int to do number compare to tell if game data is out of date or ahead?